app/models/parking_models.py

```diff
@@ -1,5 +1,5 @@
 from sqlmodel import SQLModel, Field, Relationship
-from typing import Any, Optional,List
+from typing import List, Optional
 from datetime import datetime, timezone
```

Comment:
you retained some unnecessary imports previously used in other parts of the system. Ensure that unused imports like `Any` and redundant relationships are removed to avoid confusion.